core/blockchain.go

```diff
@@ -46,7 +46,7 @@ import (
 	"github.com/eth-classic/go-ethereum/pow"
 	"github.com/eth-classic/go-ethereum/rlp"
 	"github.com/eth-classic/go-ethereum/trie"
-	"github.com/hashicorp/golang-lru"
+	lru "github.com/hashicorp/golang-lru"
```

Comment:
Uncertain if there makes a difference, but why is is that package prefaced with lru?

Reply:
just a formatting thing, since it is referenced as lru in that file, was a change made in a commit that was cherry picked into this branch

Reply:
ahh ok